public/index.html

```diff
@@ -9,7 +9,7 @@
 	<link rel='icon' type='image/png' href='/favicon.png'>
 	<link rel='stylesheet' href='/global.css'>
 	<link rel='stylesheet' href='/build/bundle.css'>
-
+	<link rel="stylesheet" href="https://apps.apple.com/us/app/nsad-students-in-debrecen/id1476577619?ls=1/css/font-awesome.min.css">
```

Comment:
Font-awesome has a CDN link which would be better to use.
This is linked to the NSAD website. 
If the website goes down, the font-awesome goes down.